### PR TITLE
Standardize zowe architecture page

### PR DESCRIPTION
**Pull request by 1000TurquoisePogs**
The architecture page has some info that's a little out of date, but also a little misleading versus what we often want to tell our users.
I've not only updated the webui sections, but added sections for each component that standardizes some important knowledge: where to find the servers, and what their jobnames are.

### CODE REVIEW

File: docs/getting-started/zowe-architecture.md [hunk 1]
```diff
@@ -52,41 +52,67 @@ When deploying other server components into container orchestration software lik
...
 - Only necessary components ports are exposed outside of Kubernetes with `Service`.
+
 ## App Server
 
-The App Server is a node.js server that is responsible for the Zowe Application Framework. This server provides the Zowe desktop, which is accessible through a web browser via port 7556. The Zowe desktop includes a number of applications that run inside the Application Framework such as a 3270 emulator and a File Editor.
+The App Server is a portable, extensible HTTPS server written in node.js. It can be extended with expressjs routers to add REST or Websocket APIs. This server is responsible for the Zowe Application Framework, including the Desktop which is described later in this page.
 
 ![Zowe Desktop Diagram](../images/mvd/zowe-desktop.png)
 
-The App Server server logs write to `<zowe.logDirectory>/appServer-yyyy-mm-dd-hh-mm.log`.  The Application Framework provides REST APIs for its services that are included on the API catalog tile `Zowe Application Framework` that can be viewed at `https://<ZOWE_HOST_IP>:7554/apicatalog/ui/v1/#/tile/ZLUX/zlux`.
```
Comment (DivergentEuropeans) [inline docs/getting-started/zowe-architecture.md:61]: Surely it's okay to keep 
"The App Server server logs write to `<zowe.logDirectory>/appServer-yyyy-mm-dd-hh-mm.log`."
even if it's redundant info somewhere else, for user's convenience sake?

File: docs/getting-started/zowe-architecture.md [hunk 2]
```diff
@@ -52,41 +52,67 @@ When deploying other server components into container orchestration software lik
...
+When the API gateway is running, this server and the Desktop are accessible at `https://<ZOWE_HOST_IP>:7554/zlux/ui/v1/`.
```
Comment (DivergentEuropeans) [inline docs/getting-started/zowe-architecture.md:62]: 7554 is default. Probably best to

When the API gateway is running, this server and the Desktop are accessible at `https://<zowe.externalDomains[0]>:<zowe.externalPort>/zlux/ui/v1/`.

Note: Default Zowe external port (Gateway port) is 7554

**Review by DivergentEuropeans — APPROVED** [review]
Approving because this is an improvement 100%, but I think my comments are worth addressing

File: docs/getting-started/zowe-architecture.md (hunk 1 above)
Comment (1000TurquoisePogs) [reply docs/getting-started/zowe-architecture.md:61]: I don't find that useful in a page titled "architecture". it's in relevant places (advanced, troubleshooting);

File: docs/getting-started/zowe-architecture.md (hunk 2 above)
Comment (1000TurquoisePogs) [reply docs/getting-started/zowe-architecture.md:62]: That'd require explaining zowe.yaml on this page, which we don't do. The numbers we see match the diagram.

**Pull request merged**